src/components/core/enums.ts

```diff
@@ -12,7 +12,5 @@ export enum DisplayLayout {
 
 export enum FeatureFlag {
   GENERAL_MEMBERSHIP_MINT = 'GENERAL_MEMBERSHIP_MINT',
-  MARKDOWN_SHORTCUTS = 'MARKDOWN_SHORTCUTS',
-  FOLLOW = 'FOLLOW',
```

Comment:
tossed these assuming these experiments have been resolved. otherwise would've been a more annoying migration to keep these around